node/src/Commands.ts

```diff
@@ -328,21 +328,34 @@ export function createMGet(keys: GlideString[]): command_request.Command {
  * @internal
  */
 export function createMSet(
-    keyValueMap: Record<string, string>,
+    keysAndValues: Record<string, GlideString> | GlideRecord<GlideString>,
```

Comment:
We discussed to do conversion before calling `createWhatever` method. Not a strict requirement though. I'm ok to forget it.

Reply:
oh! that's right.  
it was even my recommendation...